### DISCUSSION TITLE
Changing gastric pH doesn't affect the PK profile of weak base drugs with pH-dependent solubility

### DISCUSSION THREAD

**Discussion by wuershaonian** [Open-Systems-Pharmacology/Forum#1657] [Questions & Problems]
Dear All,

I am working on a weak base drug with pH-dependent solubility. It did not show any change with the change of gastric pH. But I noticed that the solubility decreased indeed with the increase of gastric pH in advanced parameter page of the simulation. Dose the setting of formulation or pKa value affect the result? I wonder why the change of gastric pH does not connect to the PK profiles.

Besides, how can I obtain the fa value in PK-sim?

Thanks a lot.

**Comment by AndreDlm** [comment]
Dear @wuershaonian, 

Formulation settings do not influence the solubility, but pKa values obviously do so. They are used to calculate the solubility at a given pH value according to the Henderson Hasselbach equation. You can visualize the solubility-pH profile of your drug in the `Compound` Building Block here:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/d7d5f301-4725-4a78-a2e9-d3e146869b82)
The solubility along the GI tract will be the same as displayed for the relevant pH values in this graph. You can find more information in the Manual [here](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow) under the section solubility. 

To get the fa, you need to select it as output under `Define Settings and Run` --> `Lumen`:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/cf95be76-f808-4216-aee2-1d863c3ee206)

**Reply by woniujy** [reply]
Hi André, I meet the same question here. The solubility was input as table function of pH in the compound. When I changed the gastric pH from 2 to 6, the PK profiles did not show any difference. And sensitivity analysis showed that gastric pH was not sensitive. I feel that there may be something wrong. Is there any relevent setting that I do not pay attention to, such as formulation? In addition, does pKa work when pH-solubility table is used? Thanks very much.

**Reply by AndreDlm** [reply]
Hi @woniujy, formulation does not have an influence on the solubility, but one explanation for your finding could be that the release rate of your formulation is so low in the stomach that the solubility limit is not reached, neither at pH 2 or 6. To test this, you could plot the luminal drug concentration in the stomach and compare it with the solubility. Without knowing more details, it's hard to comment, though. 
If this doesn't help to solve your question, it would be helpful if you could share more information on the formulation and solubility profile.

**Reply by woniujy** [reply]
Hi André, the solubility profile is pasted below. I found that with this solubility profile, when the formulation was "dissolved" or "Weibull", the PK profile did not change with GIT pH. But once the formulation "particle dissolution" was used, the PK profile varied obviously when GIT pH was changed from 2 to 6. I do not understand this and do not know which is the right result. Looking forward to your reply. Thank you. 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/131863437/86433a4c-dd15-4747-a18f-d5fdba416224)

**Reply by AndreDlm** [reply]
I think there are several pitfalls to be aware of. First, please mind that the release kinetics are probably different between the `Weibull`, `dissolved`, and `particle dissolution` model which can give rise to different PK profiles. I recommend looking at the simulated fraction dissolved and luminal concentrations in the upper GI tract to get a feeling of the different release kinetics and the PK they result in. 

Second, the solubility in the table above is very high (at least for the lower pH values), so if the dose is low, the solubility limit might not be reached. 

Lastly, the plot above generated by the software is misleading. In fact, PK-Sim interpolates linearly between the values you have defined in the table. On a linear scaled plot, this looks as follows:

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/15ced393-54cb-47c1-b9d4-c3ca7e6bc8b0)

**PK-Sim calculates with the plotted values and line shown above.** 

Unfortunately, when plotting the data on a log scale, PK-Sim connects the observed values again with lines which is incorrect (dashed red line). Instead, because it's on a log scale, an exponential curve should be drawn between the observations (black solid line):

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/3e6adc1f-54cf-4e75-bc92-5fee7f8c0b28)

In other words, when changing the stomach pH to 6, the calculated solubility is 2077 mg/L which might explain some of the confusion here.
